code/modules/reagents/chems/chems_ethanol.dm

```diff
@@ -99,6 +89,22 @@
 	if(halluci)
 		M.adjust_hallucination(halluci, halluci)
 
+// Somewhat a dummy type for 'pure ethanol' to avoid having to set dirtiness/heating products/etc on literally everything else.
+/decl/material/liquid/ethanol/spirits
+	dirtiness         = DIRTINESS_STERILE
+	uid               = "chem_ethanol"
+
+	// Uncomment when refining spirits is less annoying
```

Comment:
maybe note that it's specifically about draught/heating controls? currently they aren't finely-grained enough to allow a useful degree of control. we might look into a stopgap based on autogenerating the levels from burn milestone materials in the meantime but i think that failed to work earlier for some reason